radicale/__init__.py

```diff
@@ -92,6 +93,14 @@ def __init__(self, address, handler):
         self.server_activate()
 
 
+class ThreadedHTTPServer(socketserver.ThreadingMixIn, HTTPServer):
```

Comment:
The concern I have with `daemon_thread = True` is that partial uploads get aborted.

Perhaps atomic file writes (write to temporary file and then move) is the solution.